examples/online/index.js

```diff
@@ -1,7 +1,11 @@
-// first:
-// $ npm install redis online
+// first (if you have not done it yet):
+// $ npm install redis
+// $ brew install redis
```

Comment:
Not sure I agree with adding this.  In an example the point is to show `express`, the assumption being you already know you need a `redis` instance running somewhere.  I guess I might be fine with saying `ensure you have redis running on its default port`, since it does not assume osx and brew.